CommandLine/emake/SOG.cpp

```diff
@@ -7,54 +7,40 @@
 #include <fstream>
 #include <iostream>
 
-bool ReadSOG(const std::string &input_file, Game *game) {
-  event_parse_resourcefile();
-  map<string, evpair> event_filenames;
-  for (evpair e : event_sequence) {
-    event_filenames[event_get_function_name(e.first, e.second) + ".edl"] = e;
-    if (!e.second && event_is_instance(e.first, e.second)) {
-      // For stacked events, allow more than just the 0th instance.
-      // Needs to be big enough to accommodate the largest vk_ constant and also
-      // provide a reasonable number of alarm events, user events, etc.
-      // Using BYTE_MAX gives us both, and plenty of wiggle room.
-      for (int i = 1; i < 255; ++i) {
-        event_filenames[event_get_function_name(e.first, i) + ".edl"] = e;
-      }
-    }
-  }
+using std::map;
+using evpair = std::pair<int, int>;
```

Comment:
I would prefer a struct over std::pair<int,int> for readability but if you don't want to change itd be nice to at least comment what the ints refer to.